Applications/MNIST/res/mnist.ini

```diff
@@ -7,7 +7,7 @@ Optimizer = adam 	# Optimizer : sgd (stochastic gradien decent),
  	    		#             adam (Adamtive Moment Estimation)
 Loss = cross  		# Loss function : mse (mean squared error)
                         #                       cross ( for cross entropy )
-Save_Path = "mnist_model.bin"  	# model path to save / read
+# Save_Path = "mnist_model.bin"  	# model path to save / read
```

Comment:
Any reason for this? does not need anymore?

Reply:
From #1007, running app test more than twice will result is test fail because now we have deterministic bin path.
So, commented this line to create reproducible result every time.

Reply:
How do you plan to handle some of the apps which use their saved model files for testing? I think the draw_classification application requires that.

Reply:
if it requires saving, it has to be deleted after testing or resetted to remain determinant. Is there a test that requires that though?

Reply:
TransferLeaning/Draw_Classification does it.

Reply:
Acked, remove weights at the end of the program.

Reply:
thanks.

Reply:
> From #1007, running app test more than twice will result is test fail because now we have deterministic bin path.
> So, commented this line to create reproducible result every time.

It might be better to have saving trained model with time_stamp.

Reply:
Thank you for the suggestion. I think it's  good to have it as an argument. although saving dynamically will needs more implementation, so I'll do it as a separate PR.